spec/controllers/idv/in_person/usps_locations_controller_spec.rb

```diff
@@ -374,11 +374,20 @@
       end
 
       it 'updates proofing component vendor' do
-        expect(user.proofing_component&.document_check).to be_nil
+        proofing_components = Idv::ProofingComponents.new(
+          idv_session: controller.idv_session,
+          session: controller.session,
+          user_session: controller.user_session,
+          user:,
+        )
+
+        expect(proofing_components.document_check).to be_nil
```

Comment:
I feel like I'm missing some context here. Is an equivalent to `Idv::ProofingComponents.new` happening in the controller?

Reply:
`Idv::ProofingComponents` reads info from sessions (Rails, user, and Idv::) to derive the current components. Previously the controller was updating the proofing_component attached to the user. Here we're just taking the various sessions the controller is interacting with and essentially checking that it made the changes we are expecting. You could make an argument that this test might be duplicative now (we _should_ be testing that the controller is updating `Idv::Session` appropriately elsewhere)